### DISCUSSION TITLE
Feature Request: Disable Editing in GitHub’s Web Interface

### DISCUSSION THREAD

**Discussion by ttsc** [community/community#128523] [Repositories]
### Select Topic Area

Product Feedback

### Body

It would be very useful to have the possibility to disable editing via the web interface.

We have set up several pre-commit hooks in our repository to validate the code before each commit. However, these pre-commit hooks are skipped when committing through the GitHub web interface, leading to issues.

An ideal solution would be to include an option in the repository settings to disable the web editor.

**Comment by alxndrsn** [comment]
It would be great to be able to disable the editor per user account too - it significantly slows down page-load, and occasionally hijacks keypresses.
